.github/workflows/create-release.yml

```diff
@@ -122,6 +123,60 @@ jobs:
       - name: Run tests
         run: poetry run poe test
 
+  run-unit-tests-indexer:
+    ## **IMPORTANT**: If any changes are made to how to run the unit tests
+    ## in the release-workflow, make sure to update the steps
+    ## in the unit-test.yaml workflow as well.
+    name: Run indexer unit tests
+    needs: validate-input-params
+    runs-on: ubuntu-latest
+    permissions: read-all
+    env:
+      RELEASE_BRANCH: ${{ needs.validate-input-params.outputs.release_branch }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with: # checkout the release branch.
+          ref: ${{ needs.validate-input-params.outputs.release_branch }}
+
+      - name: Check if the checked out branch is the release branch.
+        run: |
+          git branch --show-current
+          git branch --show-current | grep -q ${RELEASE_BRANCH}
+
+      - name: Extract Python version
+        id: python-version
+        run: ./scripts/shell/extract-python-version.sh
+
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ env.PYTHON_VERSION }}
+
+      - name: Install Poetry
+        run: |
+          curl -sSL https://install.python-poetry.org | python3 -
+          echo "$HOME/.local/bin" >> $GITHUB_PATH
+
+      - name: Install dependencies
+        working-directory: ./doc_indexer
+        run: poetry install --with test
+
+      - name: Create config.json
+        run: |
+          mkdir -p config
+          echo '{"mock-key": "mock-value"}' > config/config.json
+
+      - name: Run tests
+        working-directory: ./doc_indexer
+        env:
+          LOG_LEVEL: "DEBUG"
+          # Kyma docs path.
+          DOCS_SOURCES_FILE_PATH: "./docs_sources.json"
+          DOCS_PATH: "/test-data"
+          DOCS_TABLE_NAME: "kc_pr_${{ github.event.pull_request.number }}"
```

Comment:
`github.event.pull_request.number` is not valid here. This is not a pull request.